docs/integrations/sources/google-ads-migrations.md

```diff
@@ -1,5 +1,24 @@
 # Google Ads Migration Guide
 
+## Upgrading to 3.0.0
+
+This release upgrades the Google Ads API to version 15 which causes the following changes in the schemas:
+
+| Stream                     | Current field name                                                         | New field name                                                           |
+|----------------------------|----------------------------------------------------------------------------|--------------------------------------------------------------------------|
+| ad_listing_group_criterion | ad_group_criterion.listing_group.case_value.product_bidding_category.id    | ad_group_criterion.listing_group.case_value.product_category.category_id |
+| ad_listing_group_criterion | ad_group_criterion.listing_group.case_value.product_bidding_category.level | ad_group_criterion.listing_group.case_value.product_category.level       |
+| shopping_performance_view  | segments.product_bidding_category_level1                                   | segments.product_category_level1                                         |
+| shopping_performance_view  | segments.product_bidding_category_level2                                   | segments.product_category_level2                                         |
+| shopping_performance_view  | segments.product_bidding_category_level3                                   | segments.product_category_level3                                         |
+| shopping_performance_view  | segments.product_bidding_category_level4                                   | segments.product_category_level4                                         |
+| shopping_performance_view  | segments.product_bidding_category_level5                                   | segments.product_category_level5                                         |
+| campaign                   | campaign.shopping_setting.sales_country                                    | This field has been deleted                                              |
+
+Users should:
+- Refresh the source schema
+- And reset affected streams after upgrading to ensure uninterrupted syncs.
```

Comment:
Reset affected streams after upgrading to ensure uninterrupted syncs.

Reply:
Updated